### DISCUSSION TITLE
How do I connect to Milvus EC2 Cluster after deploy and start?

### DISCUSSION THREAD

**Discussion by elspeth-strathern** [milvus-io/milvus#31288] [Q&A and General discussion] [answered]
I don't see any explanation as to which instance I should be connecting to. I found some information that I should be able to connect to any of them? But when I do I get a Timeout error. I am able to ssh in, so it is not a firewall or permissions issue.

Any help would be appreciated.

**Comment by elspeth-strathern** [comment]
Apparently it is the instance named 'proxy'. Would recommend adding this to documentation.

**Comment by yhmo** [comment, marked as answer]
The "Overview" and "access-layer" mentioned this:
https://milvus.io/docs/architecture_overview.md
https://milvus.io/docs/four_layers.md#Access-layer

![Screenshot from 2024-03-15 10-24-02](https://github.com/milvus-io/milvus/assets/2282099/2b307141-8f04-4fcb-abcc-e2620c1805f5)
